ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -522,6 +522,16 @@ CH_SS_RMM_SCHWTC_CHDXSC_CCM_CHWSWISOVPM_CWSWISOVPM_IGM_REFPM_REFSM_MIPVCM_MIPWM_
   - CWDT
   - INVOPWM
   - INVIPCM
+  
+CH_VSC_CHWFRSM_CHDXSC_CHWISOVM:
+  description: "Water cooled chiller with VFD ,supply ,return water temperature ,flow and isolation valve monitoring."
+  is_canonical: true
+  implements:
+  - CH
+  - VSC
```

Comment:
can you use `CHDXVSC` instead of `VSC`?

Reply:
Yes agreed used the CHDXVSC instead of VSC and done with the changes in recent commits